components/shared/Button.js

```diff
@@ -2,7 +2,7 @@
 
 import Link from 'next/link'
 
-import styles from '../styles/Button.module.css'
+import styles from './Button.module.css'
 
 function Button(props) {
```

Comment:
can you use a deconstructive thing here and add function documentation.

i,e `function Button({param1, param2, ....})`

Reply:
sure thing